algorithm.go

```diff
@@ -22,6 +22,10 @@ const (
 	// Requires an available crypto.SHA512.
 	AlgorithmPS512 Algorithm = -39
 
+	// RSASSA-PKCS1-v1_5 using SHA-256 by RFC 8812.
+	// Requires an available crypto.SHA256.
+	AlgorithmRS256 Algorithm = -257
```

Comment:
consider a link / comment to webauthn: https://www.w3.org/TR/webauthn-2/#sctn-sample-registration